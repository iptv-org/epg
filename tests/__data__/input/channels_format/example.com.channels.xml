<?xml version="1.0" encoding="UTF-8"?>
<channels>
  <channel site="example.com" xmltv_id="wrong_id" site_id="240" lang="en" logo="https://example.com/logo.png">BBC World News</channel>
  <channel site="example.com" lang="en" xmltv_id="CNNInternational.us" site_id="140" >CNN International</channel>
</channels>
