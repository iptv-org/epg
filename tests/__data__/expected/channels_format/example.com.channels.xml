<?xml version="1.0" encoding="UTF-8"?>
<channels>
  <channel site="example.com" site_id="140" lang="en" xmltv_id="CNNInternational.us@MENA">CNN International</channel>
  <channel site="example.com" site_id="240" lang="en" logo="https://example.com/logo.png" xmltv_id="">BBC World News</channel>
</channels>
